src/examples/basic/formattedPrice.test.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+/* eslint-disable  react/prop-types */
+// import FormattedPrice from './components/formattedPrice';
+
+
+describe('Querying for Abnormal Text', () => {
+  // There's a few different ways you can test for text
+  // That is broken across several elements
+  test('text is broken across several elements - container query', () => {
+    // Component to test is a formatted Price, where dollars and cents are split
+    const FormattedPrice = ({ price }) => {
```

Comment:
Shouldn't you import the version outside of the test?

Reply:
I am trying to decide for readability sake, is it easier to understand what you are testing if the component is colocated in the test file.

Reply:
I think it's less important when this is written up:

https://hdquotecenter.atlassian.net/wiki/spaces/TECH/pages/815169664/WIP+React+Testing+Library